simulation/traffic_simulator/src/traffic_lights/traffic_light_state.cpp

```diff
@@ -38,10 +39,9 @@ std::istream & operator>>(std::istream & is, TrafficLightColor & datum)
   try {
     datum = conversions.at(value);
   } catch (const std::out_of_range &) {
-    std::stringstream what;
-    what << "An invalid value '" << value
-         << "' was specified for type 'traffic_simulator::TrafficLightColor'.";
-    throw std::runtime_error(what.str());
+    ;
```

Comment:
Remove this line.

Reply:
fixed in https://github.com/tier4/scenario_simulator_v2/pull/306/commits/8e359cbe365195565e1bbbfbc19663f990c859c1